libs/workspace-manager/src/lib/workspace-manager/features/connect-workspace/connect-workspace.component.html

```diff
@@ -1 +1,12 @@
-<p>connect-workspace works!</p>
+<form
```

Comment:
In Generators we plan to use formly to build a form builder component that will receive all the metadata from the backend.
I know we are building now a small POC but maybe we should be aligned about how we build forms. WDYT?

Reply:
Yeah I agree, definitely but this entire code will be deleted. This is just for development and maybe for the upcoming meeting with the angular team